main/lsp/request_dispatch.cc

```diff
@@ -64,11 +321,22 @@ void LSPLoop::processRequestInternal(LSPMessage &msg) {
         auto &params = msg.asNotification().params;
         if (method == LSPMethod::SorbetWorkspaceEdit) {
             auto &editParams = get<unique_ptr<SorbetWorkspaceEditParams>>(params);
-            typecheckerCoord.typecheckOnSlowPath(move(editParams->updates));
```

Comment:
(You may be concerned that previously we did not have a `FastPathTypecheckTask`. I had that thought initially, too. Then I noticed that the previous implementation of `typecheckOnSlowPath` just calls `LSPTypechecker::typecheck`, which takes the fast path if it's a fast path update.)